src/keydra/providers/splunk_hec.py

```diff
@@ -108,14 +111,15 @@ def validate_spec(cls, spec):
         if 'rotatewith' not in spec['config']:
             return (False, "Config must contain 'rotatewith' section")
         else:
-            if not all(k in spec['config']['rotatewith'] for k in ['key', 'provider']):
+            if not all(k in spec['config']['rotatewith']
+                       for k in ['key', 'provider']):
                 return (False, "'rotatewith' must contain 'provider' and 'key'")
 
         return True, 'It is valid!'
 
     @classmethod
     def redact_result(cls, result, spec=None):
-        if 'value' in result and 'secret' in result['value']:
-            result['value']['secret'] = '***'
```

Comment:
By the looks of it, this would have leaked `hecToken`

Reply:
Good catch!